benchmark/README.md

```diff
@@ -0,0 +1,94 @@
+# Duck-Read-Cache-FS Performance Benchmark Suite
+
+## Overview
+This benchmark suite evaluates the performance of our DuckDB fs extension compared to the standard DuckDB httpfs. 
+Key aspects measured include:
+
+* Out-of-the-box parallel read support ✅
+* Read performance from cloud storage ✅
+* Compatibility with existing DuckDB httpfs workflows ✅
+* Memory usage and disk utilization ✅
+* End-to-end query execution time with cached vs. non-cached data (work in progress) 
+
+> **Note**
+> Our goal is to provide a more efficient alternative to DuckDB's httpfs extension while maintaining full compatibility. By caching the data locally, you not only get a performance boost but also save network costs from object storage.
```

Comment:
`network costs` -> `egress cost`

Reply:
I don't think it's suitable to place under benchmark readme, it should appear in the main page.